libmscore/rest.cpp

```diff
@@ -392,23 +392,22 @@ void Rest::layout()
       rxpos() = 0.0;
       const StaffType* stt = staffType();
       if (stt && stt->isTabStaff()) {
-            const StaffType* tab = stt;
```

Comment:
I don't see any necessity for this additional variable, so removed it and use the 'original' one later instead